### ISSUE TITLE
Requset: Add to HACS

### ISSUE THREAD

**Issue by JamieP83** [mtgeekman/Home_Assistant_NeoSmartBlinds#6]
Hey!
thanks so much for all you do on this!
any chance it can be added to HACS, to make it even easier please?

**Comment by mtgeekman** [comment]
It's on my list of things to do this weekend.

**Comment by JamieP83** [comment]
thank you!

**Comment by jgrn307** [comment]
Any updates on this?  I was hoping to try the top up bottom down thing but I was hoping to use HACS to install it!

**Comment by mtgeekman** [comment]
Yes, sorry. I am working through their documentation.  Had some family things come up last week and am working on getting back to putting this on HACS

**Comment by mtgeekman** [comment]
Pull request submitted to HACS to be added, will work through any issues as needed.

**Comment by jgrn307** [comment]
Any updates on this?  Cheers!

**Comment by jimmyleet** [comment]
This is a great idea, I love the addon but I always forget about it! I didn't realise how many new features are added!

**Comment by devonian79** [comment]
I was searching around for this in HACS and it didn't come up. Is it still in there?

**Comment by mtgeekman** [comment]
Currently you are able to add this repository as a "custom repository" in HACS and then install it that way. 

I would still like to get it added to HACS as a searchable option.

**Comment by ikifar2012** [comment]
> Currently you are able to add this repository as a "custom repository" in HACS and then install it that way.
> 
> I would still like to get it added to HACS as a searchable option.

@mtgeekman I have made a PR to take the first steps to adding it to HACS default repo, check #54
